### PR TITLE
[CI-SKIP] fix avoid-hopper-search obfhelper hint type

### PR DESCRIPTION
**Pull request by Cat7373**

### CODE REVIEW

**Comment by mikroskeem** [comment]
Prefix the commit message with `[CI-SKIP]`

File: Spigot-Server-Patches/0392-Avoid-hopper-searches-if-there-are-no-items.patch [hunk 1]
```diff
@@ -74,7 +74,7 @@ index 06b4dc628..37b12daff 100644
  
 +            // Paper start - Don't search for inventories if we have none, and that is all we want
 +            /*
-+             * We check if they want inventories by seeing if it is the static `IEntitySelector.c`
++             * We check if they want inventories by seeing if it is the static `IEntitySelector.d`
```
Comment (kennytv) [inline Spigot-Server-Patches/0392-Avoid-hopper-searches-if-there-are-no-items.patch:77]: every time an obfuscated methods/field is used, a puppy dies; refer to its obfhelper (`isInventory()`)

**Review by kennytv — CHANGES_REQUESTED** [review]

**Comment by BillyGalbreath** [comment]
This isn't a typo. It's the result of not using the obfhelper. Fix this to use the obfhelper, otherwise someone else is going to be fixing this again next time the mapping changes ;)

File: Spigot-Server-Patches/0392-Avoid-hopper-searches-if-there-are-no-items.patch (hunk 1 above)
Comment (electronicboy) [reply Spigot-Server-Patches/0392-Avoid-hopper-searches-if-there-are-no-items.patch:77]: This comment is pointing towards an obfhelper where the helper has been updated, but the "what this should be" is outdated, it's technically a non-issue due to the nature of the full comment, but, there is no reason why it shouldn't be resolved

**Pull request merged**